go.mod

```diff
@@ -48,7 +48,10 @@ require (
 	gopkg.in/yaml.v2 v2.4.0
 )
 
-require google.golang.org/protobuf v1.27.1
+require (
+	github.com/onsi/ginkgo v1.16.4
+	google.golang.org/protobuf v1.27.1
```

Comment:
Also, why was a second protobuf recently added in its own require block recently? If the intent is to update from
`github.com/golang/protobuf v1.5.2`
to
`google.golang.org/protobuf v1.27.1`
shouldn't that be done for the entire repo rather than add a second protobuf?

Reply:
@jt-nti I still have this question about the change in https://github.com/hyperledger/fabric/pull/3522